templates/docs/base/forms.md

```diff
@@ -219,6 +219,12 @@ When using a password field, use this pattern to allow the user to toggle the pa
 View example of the password toggle
 </a></div>
 
+## Accessibility
+
+Validated form input elements should indicate errors with `aria-invalid` attribute. JavaScript is required to achieve this.
```

Comment:
I don't think it's necessarily JS. If a form is POSTed and server-side rendered with errors, `aria-invalid` can be added to HTML on server side as well.